test/integration/production/test/index.test.js

```diff
@@ -553,6 +552,25 @@ describe('Production Usage', () => {
     expect(text).toBe('Not AMP')
   })
 
+  it('should warn when prefetch is true', async () => {
+    let browser
+    try {
+      browser = await webdriver(appPort, '/development-logs')
+      const browserLogs = await browser.log('browser')
```

Comment:
Since this is in the production suite we need to check the `global.browserName` is `chrome` before using `browser.log` since it's only supported in it currently and not the other browsers we test. Related PR: #8234